Pipelines/Templates/Sign.yaml

```diff
@@ -38,15 +38,41 @@ jobs:
       scanType: 'Register'
       verbosity: 'Verbose'
       alertWarningLevel: 'High'
-  - task: PkgESCodeSign@10
+  - task: EsrpCodeSigning@1
     displayName: 'CodeSign Binaries - ''SigningBranch'' only'
     env:
       SYSTEM_ACCESSTOKEN: $(System.AccessToken)
     inputs:
-      signConfigXml: '$(Build.SourcesDirectory)\Tools\SignConfig.xml'

```

Comment:
Should this file get deleted now?

Reply:
Yeah, good call.

Reply:
Oh darnit, I forgot about Nuget package signing. Will post an update...